### PR TITLE
liberal-crime-squad: init at 2016-03-03

### PR DESCRIPTION
**Pull request by rardiol**
###### Things done
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
Looks good to me :+1:

File: pkgs/games/liberal-crime-squad-git/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, stdenv, ncurses, autoconf, automake , SDL2, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  version = "2016-03-03";
+  name = "liberal-crime-squad-git-${version}";
```
Comment (abbradar) [inline pkgs/games/liberal-crime-squad-git/default.nix]: I don't think `-git` suffix is needed (both there and in other places), unless there would also be non-git version of this in nixpkgs.

File: pkgs/games/liberal-crime-squad-git/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    repo = "Liberal-Crime-Squad";
+    rev = "39c5f167e902e33cb37b152215d67f71b9ae8269";
+    sha256 = "18s2w2570fd79y9rb7ikxls1qw29l1lbwk9n2dxxqjslbzrbvv31";
+  };
+
+  preConfigure = ''
+    ./bootstrap
+  '';
+
+  buildInputs = [ ncurses autoconf automake SDL2 SDL2_mixer ];
```
Comment (abbradar) [inline pkgs/games/liberal-crime-squad-git/default.nix]: Maybe `autoreconfHook` (with no call to `./bootstrap`) would be cleaner, but that's of no big concern.

File: pkgs/games/liberal-crime-squad-git/default.nix (hunk 1 above)
Comment (ricardoardissone) [inline pkgs/games/liberal-crime-squad-git/default.nix]: I was following the manual: http://nixos.org/nixpkgs/manual/#sec-package-naming. It's possible a non-git version could be added, it's just that upstream doesn't seem to have released one in years.

Comment (abbradar) [inline pkgs/games/liberal-crime-squad-git/default.nix]: Hm, indeed, I haven't known that we have such a line in the manual. However, our usual practice is to name at least directories and attributes without the suffix. I filed https://github.com/NixOS/nixpkgs/issues/14572 to discuss this; meanwhile let's use your own judgement.

**Comment by abbradar** [comment]
Looks good to me; notify whether you decide to leave current package naming or remove the suffix from somewhere. I'll merge it either way (disclaimer: I've wanted to try this game out for some time ^_^).

**Comment by ricardoardissone** [comment]
Renamed without git as seems to be the consensus on #14572 and I'm doubtful about the existence of a future formally released version that would require a separate vcs and non-vcs package.

**Comment by abbradar** [comment]
Oh, two more moments: rename the commit to have no `-git` suffix and (more important) please consider adding yourself as a maintainer!

**Comment by ricardoardissone** [comment]
Renamed the commit, but I don't want to add myself as maintainer before I'm sure I will keep using NixOS (it hasn't even been a week yet).

**Comment by abbradar** [comment]
Ah, I understand then. Well, this package is simple enough that we can probably pull it without a maintainer.

**Comment by abbradar** [comment]
Thanks for our work ^_^

**Pull request merged**